src/visitpy/visitpy/PyDBOptionsAttributes_Helpers.h

```diff
@@ -11,8 +11,8 @@
 //
 // Helpers for dealing with dict reps of our DB Options
 //
-PyObject * VISITPY_API  PyDBOptionsAttributes_CreateDictionaryFromDBOptions(const DBOptionsAttributes &opts, bool show_enum_opts);
-std::string VISITPY_API PyDBOptionsAttributes_CreateDictionaryStringFromDBOptions(const DBOptionsAttributes &opts, bool show_enum_opts);
+VISITPY_API PyObject *  PyDBOptionsAttributes_CreateDictionaryFromDBOptions(const DBOptionsAttributes &opts, bool show_enum_opts);
+VISITPY_API std::string PyDBOptionsAttributes_CreateDictionaryStringFromDBOptions(const DBOptionsAttributes &opts, bool show_enum_opts);
```

Comment:
Sorry about this as well, I thought I was following an example, but I always get confused with the proper order.

Reply:
I believe putting the declaration before the function return type is always correct, but putting it after is okay if not returning a pointer. We might want to make this consistent throughout VisIt.